Objects/setobject.c

```diff
@@ -1770,7 +1770,11 @@ set_issubset(PySetObject *so, PyObject *other)
     Py_RETURN_TRUE;
 }
 
-PyDoc_STRVAR(issubset_doc, "Report whether another set contains this set.");
+PyDoc_STRVAR(issubset_doc,
+"issubset($self, /, other)\n\
+--\n\
+\n\
+Return True if all elements in this set are found in other(iterable).");
```

Comment:
Why do you use `other()` as a callable? 
```suggestion
Return True if all elements in this set are found in other iterable.");
```

Reply:
> Why do you use `other()` as a callable?

I just want to explain that the parameter `other`  is an iterable :)  Perhaps it would be better to replace `other` with `iterable`? 🤔